src/engine/client/textrender.h

```diff
@@ -162,7 +162,7 @@ class CTextRender : public IEngineTextRender
 	// support regional variant fonts
 	int m_NumVariants;
 	int m_CurrentVariant;
-	CFontLanguageVariant *m_paVariants;
+	CFontLanguageVariant *m_apVariants;
```

Comment:
`m_paVariants` or `m_pVariants` seems to be correct here.

Reply:
Yeah, @ChillerDragon could you remove the change to that variable?

Reply:
I went with `m_pVariants` since that seems the most correct. I don't understand where the `a` is coming from.

Reply:
👍